ux-microservice/.eslintrc

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
This is the Typescript config - let me work out a JS version of this and I'll send it you

Reply:
am aware, see comment below.